labelbox/schema/dataset.py

```diff
@@ -56,12 +55,6 @@ class Dataset(DbObject, Updateable, Deletable):
     row_count = Field.Int("row_count")
 
     # Relationships
-    projects = Relationship.ToMany(
```

Comment:
Can you do me a favor, so that I do not forget, add an item to a future release in CHANGELOG.

Reply:
Also, just checking: would older sdk version still work, i.e. does the backend still support this api?

Reply:
Backend has not changed, so it will return an empty list, just like before.